tests/rados/test_osd_rebalance.py

```diff
@@ -154,7 +154,7 @@ def run(ceph_cluster, **kw):
     return 0
 
 
-def wait_for_device(host, osd_id, action: str) -> bool:
+def wait_for_device_rados(host, osd_id, action: str) -> bool:
```

Comment:
Why is the old method name being changed here? 

The new method is already present here : https://github.com/harshkumarRH/cephci/blob/092c1afbf9607b7622c3761d606a23348c4a2ab1/tests/rados/rados_test_util.py#L161

Reply:
Thanks for pointing out, this was a mistake, fixed it now